packages/circuit-ui/components/ActionMenu/ActionMenu.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Status, Props, Story } from '../../../../.storybook/components';
+import * as Stories from './ActionMenu.stories';
+
+<Meta of={Stories} />
+
+# ActionMenu
+
+An ActionMenu displays a list of subsequent actions, when interacting with an actionable component.
+
+<Story of={Stories.Base} />
+<Props />
+
+- The reference element, which triggers ActionMenu, can be primary, secondary, tertiary buttons, an overflow icon, or components with embedded buttons such as the [ImageInput](Forms/ImageInput) component.
+- Each ActionMenu action item is represented by an appropriate HTML element (e.g., a button element or an anchor element).
+- If needed, the dividing line can be used to separate ActionMenu action items.
+- The leading icon is optional.
+- The ActionMenu is powered by [Floating UI](https://floating-ui.com/docs/react-dom). You can easily change the position of the ActionMenu relative to the reference element by passing in the `placement` prop. If you want to offset the ActionMenu in the x and y directions, use the `offset` prop.
```

Comment:
It's good practice to avoid subjective language such as "easy" or "simple" in documentation. Sarah Rainsberger explains this well: https://www.rainsberger.ca/blog/50-25-just-follow-this-tip/

```suggestion
- The ActionMenu is powered by [Floating UI](https://floating-ui.com/docs/react-dom). You can change the position of the ActionMenu relative to the reference element by passing in the `placement` prop. If you want to offset the ActionMenu in the x and y directions, use the `offset` prop.
```